src/components/alerts/admin/AlertsAdmin.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+
+import { useQueryClient, useMutation, useQuery } from "react-query";
+
+import { Skeleton } from "@mui/material";
+
+import FormTextField from "components/forms/FormTextField";
+
+import { useTranslation } from "i18n";
+import { Field, Form, Formik } from "formik";
+
+import {
+    allAlerts,
+    ALL_ALERTS_QUERY_KEY,
+    ACTIVE_ALERTS_QUERY_KEY,
+    addAlert,
+    removeAlert,
+} from "serviceFacades/notifications";
+
+const initialValues = {
+    startDate: "",
+    endDate: "",
+    alertText: "",
+};
+
+const AddAlertForm = ({ t, handleSubmit }) => {
+    return (
+        <Formik initialValues={initialValues} onSubmit={handleSubmit}>
+            {(props) => (
+                <Form>
+                    <Field
+                        component={FormTextField}
+                        name="alertText"
+                        required={true}
+                    />
+                    <Field
+                        component={FormTextField}
+                        name="endDate"
+                        required={true}
+                    />
+                    <Field
+                        component={FormTextField}
+                        name="startDate"
+                        required={false}
+                    />
+                </Form>
+            )}
+        </Formik>
```

Comment:
👍 
Also consider using the https://github.com/cyverse-de/sonora/blob/main/src/components/forms/FormTimestampField.js for the start and end date fields (see https://github.com/cyverse-de/sonora/blob/main/src/components/subscriptions/edit/EditSubscription.js for example).

Reply:
Ah, good! I had failed to find that before, but looks like it'll do well for that. Thanks!